install/installer/pkg/components/database/init/job.go

```diff
@@ -64,13 +88,9 @@ func job(ctx *common.RenderContext) ([]runtime.Object, error) {
 						Command: []string{
 							"sh",
 							"-c",
-							"mysql -h $DB_HOST --port $DB_PORT -u $DB_USERNAME -p$DB_PASSWORD < /db-init-scripts/init.sql",
+							fmt.Sprintf("mysql -h $DB_HOST --port $DB_PORT -u $DB_USERNAME -p$DB_PASSWORD %s< /db-init-scripts/init.sql", sslOptions),
```

Comment:
Does this need a space after the `%s`, eg.
```suggestion
							fmt.Sprintf("mysql -h $DB_HOST --port $DB_PORT -u $DB_USERNAME -p$DB_PASSWORD %s < /db-init-scripts/init.sql", sslOptions),
```

Reply:
It does not need to be there, so I'd opt for keeping the commits clean(er). :+1: 

(I wish there was a update-and-integrate-into-last-commit button here :roll_eyes: )

Reply:
> It does not need to be there, so I'd opt for keeping the commits clean(er). +1

Fine by me

> (I wish there was a update-and-integrate-into-last-commit button here roll_eyes )

Yes, that would be nice. Oh well.